### ISSUE TITLE
Not all settings are applied

### ISSUE THREAD

**Issue by marissadi** [phibr0/obsidian-latex-css#1]
I enabled Style Settings and this CSS snippet, but when I export only some of the settings are applied (e.g. I see the numbering, but the font is still the same as before). These are supposed to be applied to when I select Export to PDF, right?

**Comment by phibr0** [comment]
Is the font in the System Settings "CMU Serif"? And is everything toggled on? Sometimes the Font takes some time to load and you will need to export the note 2 times.

**Comment by marissadi** [comment]
Yes, I have toggled everything on. I have exported it a couple of times now and it is still using the default font.

![image](https://user-images.githubusercontent.com/21235875/133321683-9ac4ef29-79b9-4315-9e09-0640f2936b9f.png)

![image](https://user-images.githubusercontent.com/21235875/133321754-fbf075c6-2402-4211-b8c5-2f392c9e1301.png)

![image](https://user-images.githubusercontent.com/21235875/133322128-48c50be3-026e-4704-9e09-41bb08d6f4f8.png)

**Comment by phibr0** [comment]
Could you disable all other Snippets and Themes as a Test please

**Comment by ChenghaoMou** [comment]
Same here. I disabled all themes and plugins (except style settings) and installed CMU serif manually. The exported PDF still uses the default font.

**Comment by phibr0** [comment]
I cannot reproduce this, not even in a completely clean Vault. 
Could you try this in a newly created Vault with the Snippet and the Style Settings installed please?

**Comment by ChenghaoMou** [comment]
I created a new vault and turned on this snippet, but still, it shows the default font.

macOS 12.0 Beta (21A5506j)
Obsidian 0.12.16

**Comment by phibr0** [comment]
Is the Style Settings Plugin enabled?

**Comment by ChenghaoMou** [comment]
I think I know why. The font name installed is not `CMU Serif`, it should be `CMU Serif Roman` in my case.
![image](https://user-images.githubusercontent.com/32014649/133896699-0d56199d-68d2-4d27-8286-574a314a9125.png)

**Comment by ChenghaoMou** [comment]
The font is different after changing in style settings, but it doesn't look like `CMU Serif *` to me
![image](https://user-images.githubusercontent.com/32014649/133897035-06210d65-4c6d-420b-b963-cfffc1f1b8d8.png)

**Comment by ChenghaoMou** [comment]
They are definitely different 
![image](https://user-images.githubusercontent.com/32014649/133897352-f18cec15-9b1f-454c-ab8e-f619496f4979.png)

**Comment by ChenghaoMou** [comment]
After manually change the snippet to the following, the PDF is properly rendered with the correct font.
```css
@font-face {
  font-family: "CMU Serif Roman";
  src: local('CMU Serif Roman');
  font-weight: 500;
  font-style: normal;
  font-display: swap;
}
```

**Comment by phibr0** [comment]
This works with your locally installed Font, the Snippet ships with its own Font though and the Name is set as "CMU Serif", works on Windows and Linux atleast.

This might be a mac Issue then?

**Comment by ChenghaoMou** [comment]
I would say so. It is the only thing I changed. Changing it to other values (url, data-uri) doesn't seem to load.

**Comment by marissadi** [comment]
How are we meant to install the fonts or other things in the snippet? I just have the `latex.css` file in the `.obsidian/snippets` folder.

**Comment by phibr0** [comment]
The Font is included in the latex.css File. There is no need to install it other than putting it into the snippets folder and enabling it

**Comment by thryba** [comment]
I do have the same problem right here on a windows machine

**Comment by phibr0** [comment]
Which Theme are you using as a baseline? Please make sure to enable the Style Settings Plugin and try if it works with the default Theme.

**Comment by thryba** [comment]
At first I was using the "Sanctum" Theme, but switched backed to Default for testing. It seems to just use the default font of the theme, regardless. Style Settings Plugin is enabled and everything is shown as it should.

**Comment by sare000** [comment]
Hello - I am having similar issues. I think it may be an issue with installation. I have:
-Downloaded and enabled Style Settings Plugin
-Downloaded and put latex.css into the snippets folder

Not sure where I should put the extracted .zip file (obsidian-latex-css) though... Thanks!

**Comment by phibr0** [comment]
You don't need the zip, GitHub adds this one automatically.

**Comment by sare000** [comment]
OK thanks. Unfortunately I'm also getting the default markdown style for PDFs. If you have any suggestions, please let me know. Not sure if this is a mac issue.

**Comment by benf2004** [comment]
Also having issues with this

**Comment by vortex73** [comment]
I'm on arch linux and I've imported your release css onto my obsidian client. But my exports still have the default format. Is your source link broken for the fonts?

**Comment by benf2004** [comment]
@vortex73 check out https://github.com/benf2004/Obsidian-LaTeX-Theme
